Druid/DRSmallIntegerPrimitiveCustomisation.class.st

```diff
@@ -19,9 +19,7 @@ DRSmallIntegerPrimitiveCustomisation >> generateCustomisedPreamble: aGeneratorMe
 { #category : #'generation-IR' }
 DRSmallIntegerPrimitiveCustomisation >> initialConstraint [
 
-	^ DRRegisterDomain
-		  equals: (DRPhysicalGeneralPurposeRegister name: 'ReceiverResultReg')
-		  than: 1
+	^ DRReceiverConstraint with: (DREqualsConstraint withValue: 1)
```

Comment:
Mmmm here you are changing a Domain by a Constraint, that is not good. A domain is a tuple of Register -> Constraint.